### PR TITLE
Fixed e2e for multiple addresses on multiple flows

### PR DESCRIPTION
**Pull request by robinpiets**
Test with multipe addresses the visiblity of the restrictions on the page

Still WIP, but maybe you can already review?

### CODE REVIEW

File: packages/e2e/puppeteer/tests/location-component.ts [hunk 1]
```diff
@@ -1,54 +1,159 @@
...
+  MUNICIPAL_MONUMENT,
+  NO_CITY_SCAPE,
+  MUNICIPAL_CITY_SCAPE,
+  NATIONAL_CITY_SCAPE,
+  uniqueFixtures,
+} from "vergunningcheck-mocking/src/restriction";
 
+// Disable the warning "MaxListenersExceededWarning: Possible EventEmitter memory leak detected"
+// See: https://github.com/puppeteer/puppeteer/issues/594
+process.setMaxListeners(Infinity);
```
Comment (afjlambert) [inline packages/e2e/puppeteer/tests/location-component.ts:19]: 😱

File: packages/e2e/puppeteer/tests/location-component.ts [hunk 2]
```diff
@@ -1,54 +1,159 @@
...
+
+// Test the flow with a random address to see if all checkers are working
 const addresses = getFixturesByProperties([NO_MONUMENT, NO_CITY_SCAPE]);
-export const address = random(addresses);
-if (!address) {
-  throw new Error('address not found');
+export const randomAddress = random(addresses);
+if (!randomAddress) {
+  throw new Error("address not found");
```
Comment (afjlambert) [inline packages/e2e/puppeteer/tests/location-component.ts:25]: 👍

File: packages/e2e/puppeteer/tests/location-component.ts [hunk 3]
```diff
@@ -1,54 +1,159 @@
...
   await page.type('[name="postalCode"]', address[0]);
   await page.type('[name="houseNumberFull"]', address[1].toString());
 
-  await page.waitForSelector('[data-testid="location-found"]', { visible: true });
+  await page.waitForSelector('[data-testid="location-found"]', {
+    visible: true,
+  });
+
+  const momumentType = address[2][0];
+  const cityScapeType = address[2][1];
```
Comment (afjlambert) [inline packages/e2e/puppeteer/tests/location-component.ts]: This is very error-prone. We need to have an alternative for this. Let's discuss offline.

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: packages/e2e/puppeteer/tests/location-component.ts (hunk 3 above)
Comment (robinpiets) [reply packages/e2e/puppeteer/tests/location-component.ts]: Fixed

**Review by afjlambert — APPROVED** [review]

**Pull request merged**